File: QuickSort_Java/acmShyryiHW26_1/src/main/java/com/example/acmshyryihw26_1/ShyryiApplication.java
package com.example.acmshyryihw26_1;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.io.IOException;

public class ShyryiApplication extends Application {

    @Override
    public void start(Stage stage) throws IOException {
        FXMLLoader fxmlLoader = new FXMLLoader(ShyryiApplication
                .class.getResource("acmshyryihw26_1-view.fxml"));
        Scene scene = new Scene(fxmlLoader.load(), 510, 250);
        stage.setTitle("Ширий Богдан. " +
                "АМО Домашнє завдання №26.1, варіант №24");
        stage.setScene(scene);
        stage.show();
    }

    public static void main(String[] args) {
        launch();
    }
}

File: QuickSort_Kotlin/acmShyryiHW26_2/src/main/kotlin/com/example/acmshyryihw26_2/ShyryiApplication.kt
package com.example.acmshyryihw26_2

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Scene
import javafx.stage.Stage

class ShyryiApplication : Application() {
    override fun start(stage: Stage) {
        val fxmlLoader = FXMLLoader(ShyryiApplication
        ::class.java.getResource("acmshyryihw26_2-view.fxml"))
        val scene = Scene(fxmlLoader.load(), 510.0, 250.0)
        stage.title = "Ширий Богдан. " +
                "АМО Домашнє завдання №26.2, варіант №24"
        stage.scene = scene
        stage.show()
    }
}

fun main() {
    Application.launch(ShyryiApplication::class.java)
}